development/kops/eks-d.tpl

```diff
@@ -111,7 +110,7 @@ metadata:
   name: control-plane-{{.awsRegion}}a
 spec:
   image: 099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20201026
-  machineType: t3.medium
+  machineType: m5.xlarge
```

Comment:
We probably don't want to check this in, but the other two changes look good.

Reply:
Watching the run, not sure this is making much difference, Ill probably revert this or make it configurable if neccessary.